Source/CombatExtended/CombatExtended/ArmorUtilityCE.cs

```diff
@@ -286,7 +286,7 @@ private static float GetPenetrationValue(DamageInfo dinfo)
                     if (instigatorPawn.def == dinfo.Weapon)
                     {
                         // meleeVerbs: all verbs considered "melee worthy"
-                        Verb availableVerb = instigatorPawn.meleeVerbs.TryGetMeleeVerb();
+                        Verb availableVerb = instigatorPawn.meleeVerbs.TryGetMeleeVerb(dinfo.IntendedTarget);

```

Comment:
Do I get the correct penetration info from the dinfo? (As in vanilla or CE)
If so, I could do a refactor.

Reply:
Commented the method out, and replaced calls accordingly, see commit for more info.

Reply:
Yeah, idea is we don't need to infer penetration through obtuse hacks since we can now pass it directly via the damage info from projectile -> armor system